src/keboola_mcp_server/client.py

```diff
@@ -4,7 +4,7 @@
 import os
 import tempfile
 
-from typing import Annotated, Any, Dict, Mapping, Optional, cast, List
+from typing import Any, Dict, Mapping, Optional, cast, List
```

Comment:
While you're at it, don't use Dict/List from typing, use just regular `dict` or `list`